### PR TITLE
Added missing trainers.

### PR DESCRIPTION
**Pull request by bdebaere**
Fixes startup db errors complaining about trainer_greeting with non-existent creature. The alternative is to delete the trainer_greeting records, however this approach keeps the classic database synchronized with tbc and wotlk.

https://www.wowhead.com/npc=5032/world-alchemy-trainer
https://www.wowhead.com/npc=5037/world-engineering-trainer
https://www.wowhead.com/npc=5038/world-enchanting-trainer
https://www.wowhead.com/npc=5040/world-leatherworking-trainer
https://www.wowhead.com/npc=5041/world-tailoring-trainer

### CODE REVIEW

File: Updates/2188_missing_trainers.sql [hunk 1]
```diff
@@ -0,0 +1,6 @@
...
...`, `PowerMultiplier`, `DamageMultiplier`, `DamageVariance`, `ArmorMultiplier`, `ExperienceMultiplier`, `MinLevelHealth`, `MaxLevelHealth`, `MinLevelMana`, `MaxLevelMana`, `MinMeleeDmg`, `MaxMeleeDmg`, `MinRangedDmg`, `MaxRangedDmg`, `Armor`, `MeleeAttackPower`, `RangedAttackPower`, `MeleeBaseAttackTime`, `RangedBaseAttackTime`, `DamageSchool`, `MinLootGold`, `MaxLootGold`, `LootId`, `PickpocketLootId`, `SkinningLootId`, `KillCredit1`, `KillCredit2`, `MechanicImmuneMask`, `SchoolImmuneMask`, `ResistanceHoly`, `ResistanceFire`, `ResistanceNature`, `ResistanceFrost`, `ResistanceShadow`, `ResistanceArcane`, `PetSpellDataId`, `MovementType`, `TrainerType`, `TrainerSpell`, `TrainerClass`, `TrainerRace`, `TrainerTemplateId`, `VendorTemplateId`, `EquipmentTemplateId`, `GossipMenuId`, `AIName`, `ScriptName`) VALUES('5041','World Tailoring Trainer','Tailoring Trainer','10','10','2575','0','0','0','35','35','1','0','7','3','3','0','16','0','0','0','0','1.2','1.14286','20','0','0','0','0','1','0','1','1','1','1','1','1','5000','5000','0','0','2','2','0','0','15','1','0','2000','2000','0','0','0','0','0','0','0','0','0','0','0','0','0','0','0','0','0','0','0','0','0','0','0','0','0','0','','');
```
Comment (killerwife) [inline Updates/2188_missing_trainers.sql]: If I can be a little picky, I would suggest INSERT on first line, and then only Data on subsequent lines. Its easier to read.
Insert into bla values
(),
(),
(),...

Comment (bdebaere) [reply Updates/2188_missing_trainers.sql]: There ya go.

**Comment by bdebaere** [comment]
Is it normal that npc_trainer_template is empty on classic? Because now I'm getting a warning during startup that it does not have an entry 1000.

**Comment by cala** [comment]
Merged in https://github.com/cmangos/classic-db/commit/6c5ecd5e153c5e8cfe49e4da88676b86af59e7d8
Thank you !

**Pull request closed without merging**